tests/gluster_cli_test.py

```diff
@@ -1287,6 +1287,14 @@ def test_parseGeoRepStatus(self):
         status = gcli._parseGeoRepStatus(tree)
         self.assertEqual(status, glusterTestData.GLUSTER_GEOREP_STATUS)
 
+    def test_parseGeoRepStatusOld(self):
+        with open("glusterGeoRepStatusOld.xml") as f:
+            out = f.read()
+        tree = etree.fromstring(out)
+        gcli._TIME_ZONE = 'IST'
+        status = gcli._parseGeoRepStatus(tree)
+        self.assertEqual(status, glusterTestData.GLUSTER_GEOREP_STATUS_OLD)
```

Comment:
This is the same as the `test_parseGeoRepStatus`, except for the filename and the expected data, which could be parametrized and remove the extra test.

However, the test code is simple enough, so we can leave it as is.